src/app/RootStore.tsx

```diff
@@ -1,12 +1,15 @@
 import { NavigationStore } from '@features/home/NavigationList/model/navigation.store';
 import { UserStore } from '@entities/user/model/user.store';
+import { SidebarStore } from '@widgets/Sidebar/model/sidebar.store';
```

Comment:
```suggestion
import { SidebarStore } from '@widgets/Sidebar';
```

Reply:
impossible to do because of the import cycle problem